# Goal
geometry_msgs/PoseStamped target_pose
---
# Result
bool success
string message
---
# Feedback
geometry_msgs/PoseStamped current_pose
float32 distance_to_goal
